### PR TITLE
Added azure meetup frankfurt

### PR DESCRIPTION
**Pull request by ddobric**
:)

### CODE REVIEW

File: 2022/AzureMeetupFrankfurt/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# DotNetFriday
+
+Visit our website at https://dotnetfriday.nl
+
+![DotNetFriday](dnf.png)
```
Comment (noopman) [inline 2022/AzureMeetupFrankfurt/index.md:5]: This file reference does not work. Should it be dotnetfriday.png perhaps?

**Review by noopman — CHANGES_REQUESTED** [review]

**Comment by noopman** [comment]
Nice to see your face by the way @ddobric ! :D

**Comment by ddobric** [comment]
will correct

**Comment by davidjrh** [comment]
Hi @ddobric , seems there is an issue with your Sessionize API endpoint. Is currently returning a javascript response instead of the expected JSON when browsing https://sessionize.com/api/v2/pslfsnw6/view/Sessions, so the sessions are not being populated on the Global website.

Can you recheck the instructions at https://blog.globalazure.net/Blog/Post/284/Sessionize-integration-setup-for-local-event-organizers-for-2022-edition and that when browsing the Sessions List URL:
* ensure the result is in JSON format (if not, review the settings mentioned in the previous section)
* ensure the "startsAt" and "endsAt" are populated with date values between May 5th and May 7th

**Comment by ddobric** [comment]
Now it seems to deliver the JSON.

**Comment by davidjrh** [comment]
Hi again @ddobric, can you doble check again the setup and review the parameters? The startsAt and EndsAt fields are coming as null. Ensure you have setup the sessions on the Sessionize scheduler to allow these fields being populated.

**Pull request closed without merging**